data/match_results.json

```diff
@@ -0,0 +1,146 @@
+{
+  "match_2": {
```

Comment:
It seems to me that it doesn't meet our data structure agreement. I mean using an array instead of an object. Also I have one more question. Should we "initialize" our file with empty values? Or is it just example?

Reply:
You are right, this is just an example